src/mame/heathzenith/z100.cpp

```diff
@@ -363,8 +368,8 @@ void z100_state::machine_start()
 	save_item(NAME(m_start_addr));
 	save_item(NAME(m_vidint_enable));
 	save_item(NAME(m_memory_ctrl));
-
-	m_parity = make_unique_clear<u32[]>(m_ram.bytes() / 32);
+	save_pointer(NAME(m_gvram), 0x30000);
```

Comment:
You should register this for saving in the same function where it’s allocated.  If you don’t need to separately override `video_start` in different derived classes, you can get rid of it altogether and allocate everything here in `machine_start`.